plugins/auth-jwt/src/templates/jwt.strategy.template.ts

```diff
@@ -1,14 +1,12 @@
 import { Inject, Injectable } from "@nestjs/common";
 import { JWT_SECRET_KEY } from "../../constants";
-import { UserService } from "../../user/user.service";
 import { JwtStrategyBase } from "./base/jwt.strategy.base";
 
+declare class ENTITY_SERVICE {}
+
 @Injectable()
 export class JwtStrategy extends JwtStrategyBase {
-  constructor(
-    protected readonly userService: UserService,
-    @Inject(JWT_SECRET_KEY) secretOrKey: string
-  ) {
-    super(userService, secretOrKey);
+  constructor(@Inject(JWT_SECRET_KEY) secretOrKey: string) {
```

Comment:
Don't we need to inject the auth entity service?

Reply:
@abrl91  Yes, it's implemented in create-jwt-strategy  

<img width="648" alt="image" src="https://github.com/amplication/plugins/assets/97830649/11f8f8e5-fea9-49c6-bdd2-7e7a68c9475f">